charts/gitops-runtime/templates/_components/cap-app-proxy/environment-variables/_main-container.yaml

```diff
@@ -40,6 +40,10 @@ CORS:
       name: cap-app-proxy-cm
       key: cors
       optional: true
+DEPLOYMENT_NAME:
+  valueFrom:
+    fieldRef:
+      fieldPath: metadata.name
```

Comment:
This would give you the pod name, not the deployment name. I'm not sure how you'd get the deployment name

Reply:
oh, right. thanks for noticing. 🤦🏻 
i'll think of some other way to get it to work

Reply:
@noam-codefresh You can use this `{{ include "cap-app-proxy.fullname" . }}` This template is what defines the deployment name in the first place.